packages/api-explorer/__tests__/AuthBox.test.jsx

```diff
@@ -19,9 +19,7 @@ const props = {
 };
 
 test('should not display if no auth', () => {
-  expect(mountWithIntl(<AuthBox {...props} operation={oas.operation('/no-auth', 'post')} />).html()).toBe(
-    null,
-  );
+  expect(mountWithIntl(<AuthBox {...props} operation={oas.operation('/no-auth', 'post')} />).html()).toEqual('')
```

Comment:
Why from null to empty string? We should change the test assertion, is rendered empty string

Reply:
".html()" does not return null, as defined in the docs ( https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/html.html ), in fact (after that we deleted the yarn.lock) this test failed due to expected ".toBe(null)" and received "empty string".

Maybe an old version of enzyme returns null